gateway-service/src/test/java/org/zowe/apiml/gateway/error/check/SecurityErrorCheckTest.java

```diff
@@ -114,7 +119,60 @@ void shouldReturnCauseMessageWhenBadCredentialsException() {
         assertNotNull(actualResponse.getBody());
         List<ApiMessage> actualMessageList = actualResponse.getBody().getMessages();
         assertThat(actualMessageList, hasItem(new ApiMessage("org.zowe.apiml.security.login.invalidCredentials",
-            MessageType.ERROR, "ZWEAG120E", "Invalid username or password for URL 'null'", "Provide a valid username and password.", "The username and/or password are invalid.")));
+                MessageType.ERROR, "ZWEAG120E",
+                "Invalid username or password for URL 'null'",
+                "Provide a valid username and password.",
+                "The username and/or password are invalid."
+        )));
+    }
+
+    @Test
+    void shouldReturnCauseMessageWhenSafIdtAuthException() {
```

Comment:
Not following `given_when_then` - is this a chance to refactor this test class?

Reply:
I am just trying to split different PR, and as we approved it there. 
I would rather keep the format, even though I disagree with it.